frontend/src/component/Search/CheckboxFilter.jsx

```diff
@@ -26,15 +26,15 @@ export const CheckboxFilter = props => {
       </div>
 
       {props.showFilter && props.filterList.map(item =>
-        <div className='checkboxFilter__checkbox' key={`item__${item.value}`}>
+        <div className='checkboxFilter__checkbox' key={`item__${item.id || item.value}`}>
           <Checkbox
-            name={item.value}
-            onClickCheckbox={() => props.onChangeSearchFacets(item.value)}
-            checked={props.appliedFilterList.find(filter => filter.value === item.value)}
+            name={item.id || item.value}
+            onClickCheckbox={() => props.onChangeSearchFacets(item.id || item.value)}
+            checked={props.appliedFilterList.find(filter => filter.id ? filter.id === item.id : filter.value === item.value)}
```

Comment:
why is the need to check both?
shouldn't the data passed to this component be formatted to match a single usage?
That means that, if this is really necessary, it should be done at a higher level

Reply:
Some filters has label !== id, others no. I will fix to have a clear label and id props, even if their are the same for the filter.